samples/subsys/usb/uvc/README.rst

```diff
@@ -0,0 +1,193 @@
+.. zephyr:code-sample:: uvc
+   :name: USB Video sample
+   :relevant-api: usbd_api video_interface
+
+   USB Video sample sending frames over USB.
+
+Overview
+********
+
+This sample demonstrates how to use an USB Video Class instance to
+send video data over USB.
+
+Upon connection, a video interface would show-up in the operating system,
+using the same protocol as most webcams, and be detected as such.
+
+Any software application to read and process video would then be able
+to access the stream.
+
+Requirements
+************
+
+USB is the only requirement for this sensor, as an emulated image
+sensor and MIPI receiver is used to provide a test pattern from
+software.
```

Comment:
It describes how the sample should be, but it is only applies to certain boards.
This sample should build and work with any board with a USB device controller. The app.overlay and video-emul.overlay should be merged to app.overlay. There should be no board configurations or device tree overlays, especially not for those boards with serial backend handicap, they just have to wait for the new stack to become default.